app/models/package_manager/base.rb

```diff
@@ -105,6 +105,10 @@ def self.update(name, sync_version: :all)
 
       begin
         db_project.save!
+      rescue ActiveRecord::StatementInvalid => e
+        db_project.description = db_project.description.force_encoding('GB18030').encode('UTF-8')
+        db_project.save!
```

Comment:
@recurator are you absolutely sure this will pass? Maybe it makes sense to use Ruby's retry keyword. For example: https://www.honeybadger.io/blog/how-to-try-again-when-exceptions-happen-in-ruby/